linalg/blockmatrix.cpp

```diff
@@ -155,12 +155,11 @@ const double& BlockMatrix::Elem (int i, int j) const
    findGlobalRow(i, iblock, iloc);
    findGlobalCol(j, jblock, jloc);
 
-   if (IsZeroBlock(i, j))
+   if (IsZeroBlock(iblock, jblock))
    {
-      mfem_error("BlockMatrix::Elem");
+      return 0;
```

Comment:
This should return `zero`, with `static const double zero = 0.0;`

Reply:
Sorry for that. I pushed the correction